vllm/distributed/parallel_state.py

```diff
@@ -125,6 +129,7 @@ class GroupCoordinator:
     pynccl_comm: Optional[Any]  # PyNccl communicator
     ca_comm: Optional[Any]  # Custom allreduce communicator
     mq_broadcaster: Optional[Any]  # shared memory broadcaster
+    use_xla: bool  # Whether to use PyTorch XLA communicator
```

Comment:
does tpu platform support NCCL? if not, creating these communicators might lead to error.

Reply:
TPU doesn't support NCCL, but I didn't see any error with the other communicators.

Reply:
The TPU backend uses gloo backend in addition to the distributed runtime in `xm`. Maybe that's the reason.